apps/main/e2e/orders.spec.ts

```
@@ -0,0 +1,40 @@
+import { expect } from '@playwright/test';
+import { authenticatedTest as test, getStoreVariant } from './fixtures';
+
+/**
+ * Orders page (pro-only).
+ */
+test.describe('Orders Page (Pro)', () => {
+	test.beforeEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)